README.md

```diff
@@ -186,9 +176,7 @@ Even though computed can use state and props, they don't have to:
 }
 ```
 
-### `injectState(Component) => Component`
-
-Makes
+> Note: Computed values cannot access themselves, otherwise `Reaclette` will throw.
```

Comment:
What's the point of documenting this?

It does not makes sense for a computed to access itself, the fact that XO throws is not a feature per se, it's to help make this misuse clearer.